src/main/java/minicraft/level/tile/Tile.java

```diff
@@ -64,6 +64,9 @@ public int getDefaultData() {
 
 	/** Render method, used in sub-classes */
 	public void render(Screen screen, Level level, int x, int y) {
+		if (sprite == null) {
+			System.out.println(this);
```

Comment:
Replace with logger, but won't this spam the log? Maybe track if it has been logged and only do it once.

Reply:
Idk what this is, most likely @BenCheung0422

Reply:
Oh, I forgot removing that. That was used for debugging.